sklearn/preprocessing/label.py

```diff
@@ -661,6 +668,20 @@ class MultiLabelBinarizer(BaseEstimator, TransformerMixin):
     >>> list(mlb.classes_)
     ['comedy', 'sci-fi', 'thriller']
 
+    Perform a one-hot encoding for y labels
```

Comment:
I still find this a bit of an obscure example to be presenting for `MultiLabelBinarizer`. Its job is to deal with multilabel data, and to present a hacky use of it here seems inappropriate.